roles/openshift_logging_elasticsearch/tasks/generate_secret.yaml

```diff
@@ -4,7 +4,7 @@
   oc_secret:
     state: present
     name: "logging-elasticsearch"
-    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    namespace: "{{ openshift_logging_namespace }}"
```

Comment:
what's the reason for these namespace changes?  I thought `openshift_logging_elasticsearch_namespace` is `openshift_logging_namespace` unless you explicitly set it?

Reply:
Its set explicitly when installing [1] so when we regenerate certs it is always `openshift-logging` due to `defaults/main.yaml`.

I wasn't sure the order of resolution if we update the default value and this matches the pattern we have for kibana and curator currently.

Alternatively I could update the other components to use their specific namespace variables and then set all the namespace variables as we call `generate_secret.yaml` [2].
I'm comfortable with that change as well.

[1] https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_logging/tasks/install_logging.yaml#L89

[2] https://github.com/openshift/openshift-ansible/blob/release-3.11/playbooks/openshift-logging/private/redeploy-certificates.yml#L68

Reply:
ok - so `openshift_logging_elasticsearch_namespace` isn't set as a global default in `defaults/main.yml` it is set as a passed in role "parameter" - then we don't really allow users to override `openshift_logging_elasticsearch_namespace` for logging - then ack